### PR TITLE
Use plone i18n domain

### PR DESCRIPTION
**Pull request by gforcada**
plone.app.event is an official Plone core package,
thus their translations belong to plone.app.locales.

This commit removes the plone.app.event domain and changes it for
plone.

This fixes:
https://github.com/plone/plone.app.event/issues/181

@vincentfretin now I see the steps that have to be done (on #181) to integrate a package translations on p.a.locales

What's the best timing to do so? First merge the pull request and then do the other steps or all at once?

### CODE REVIEW

**Comment by thet** [comment]
+1, tnx!
you'd need to rename/merge all the local files too.
second step would be to merge them into plone.app.locales. only concern is, that removing the local files from here would break backwards compatibility with Plone 4.3, right?

**Comment by vincentfretin** [comment]
refs https://github.com/plone/Products.CMFPlone/issues/983

**Comment by vincentfretin** [comment]
Is master compatible with Plone 4.3? If no, we can merge and proceed with the next steps.

**Comment by thet** [comment]
yes, it is...

**Comment by vincentfretin** [comment]
In this case, it's better to keep it separated for now.

**Comment by gforcada** [comment]
@thet @vincentfretin now I think this is finally ready:
- translations are merged to plone.app.locales in https://github.com/collective/plone.app.locales/pull/195
- buildout.coredev 5.1 and 5.2 are updated in https://github.com/collective/buildout.coredev/pull/427 and https://github.com/collective/buildout.coredev/pull/428
- in this pull request all translations are removed and the domain us updated to ``plone``.

According to the guidelines in https://github.com/plone/Products.CMFPlone/issues/983 I think I did everything needed.

File: CHANGES.rst [hunk 1]
```diff
@@ -17,6 +17,8 @@ Bug fixes:
 - Fix future_events French translation.
   [vincentfretin]
 
+- Use plone i18n domain.
+  [gforcada]
```
Comment (thet) [inline CHANGES.rst]: Please move this Changelog entry to the "Breaking changes" section. Agree?

File: plone/app/event/profiles/default/actions.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
    </property>
    <property name="visible">True</property>
   </object>
-  <object name="ical_import_disable" meta_type="CMF Action" i18n:domain="plone.app.event">
+  <object name="ical_import_disable" meta_type="CMF Action" i18n:domain="plone">
    <property name="title" i18n:translate="title_ical_import_disable">Disable icalendar import</property>
```
Comment (vincentfretin) [inline plone/app/event/profiles/default/actions.xml]: Don't you need an upgrade step to change the domain of CMF Action?

Comment (gforcada) [reply plone/app/event/profiles/default/actions.xml]: Good point, I guess so. I will add it later, thanks for the review!

Comment (thet) [reply plone/app/event/profiles/default/actions.xml]: +1

**Pull request merged**